### DISCUSSION TITLE
Include the playground function in example folder :D

### DISCUSSION THREAD

**Discussion by heoyongun** [mistic100/Photo-Sphere-Viewer#793] [General]
Thanks for contributing to a great project.

I want to include a playground in my mini-project features.

Can the playground project be a public project?

**Comment by mistic100** [comment]
You are free to copy it in your project. I will not externalize it.

https://github.com/mistic100/Photo-Sphere-Viewer/blob/dev/docs/.vuepress/components/Playground.vue
